apps/src/lib/tools/jsinterpreter/CustomMarshalingInterpreter.js

```diff
@@ -408,14 +408,14 @@ module.exports = class CustomMarshalingInterpreter extends Interpreter {
     // The event loop pauses the interpreter until the native async function
     // `currentCallback` returns a value. The value contains the name of the event
     // to call, and any arguments.
-    const eventLoop = ';while(true){var event=wait();setReturnValue(this[event.name].apply(null,event.args));}';
+    const eventLoop = ';while(true){var _e=_w();setReturnValue(this[_e.name].apply(null,_e.args));}';
```

Comment:
Were these renamed to avoid collisions with the `event` parameters you added above? Any chance they can be called `_event` and `_wait` instead to be a bit more readable?